graphql/app.js

```diff
@@ -58,8 +58,8 @@ const secureCookie = process.env.HTTPS === 'true'
 // When using 'strict' value, cookies will not be sent along with requests initiated by third-party websites.
 // Reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite
 const allowedOrigins = (origin ?? '').split(' ').filter(v => v).map(o => new RegExp('^' + o))
-// Note that SameSite=none requires secure
-const sameSiteCookies = allowedOrigins.length === 1 ? 'strict' : secureCookie ? 'none' : 'lax'
+const sameSiteCookies = zoteroAuthorizeEndpoint || oicAuthUrl ? 'lax' : 'strict'
```

Comment:
Est-ce que tu te souviens pourquoi on avait mis en place cette condition `secureCookie ? 'none' : 'lax'` ?

Reply:
Quand on est en localhost, on tape sur des domaines différents (couple hôte + port) pour l'API et tout. Du coup on met à `none` pour du cross domain. C'est pour le dév uniquement.

On utilisait `strict` ou `lax` selon qu'on utilisait des frontends différents pour le back. Et là on découvre une autre limite au `strict`, ce qui fait que `lax` devient le choix par défaut, sauf en dév.

Reply:
OK, dans ce cas il faudrait plutôt faire:

```suggestion
const sameSiteCookies = process.env.NODE_ENV === 'production' ? 'lax' : 'none'
```

Il me semble qu'en développement, `oicAuthUrl` et `zoteroAuthorizeEndpoint` seront renseignés. Le mode `strict` étant trop restrictif, on est soit en mode `lax` soit en mode `none`.

Reply:
Tout à fait, bien vu. Merci !